{
  "backup_data_uid": "7ccd4b931f4b9db9",
  "backup_module_uid": "707ccdfe444cafac",
  "backup_module_uoa": "platform",
  "control": {
    "engine": "CK",
    "iso_datetime": "2020-11-14T21:43:55.245771",
    "version": [
      "1",
      "17",
      "0",
      "1"
    ]
  },
  "data_name": "ZTE K88"
}
